src/acp/policy.ts

```diff
@@ -32,18 +32,26 @@ import {
   createSolidDataset,
   saveSolidDatasetAt,
 } from "../resource/solidDataset";
+import { getUrl, getUrlAll } from "../thing/get";
 import { setUrl } from "../thing/set";
-import { createThing, getThing, setThing } from "../thing/thing";
+import {
+  createThing,
+  getThing,
+  getThingAll,
+  isThingLocal,
+  removeThing,
+  setThing,
+} from "../thing/thing";
 
 export type PolicyDataset = SolidDataset;
-export type AccessPolicy = Thing;
+export type Policy = ThingPersisted;
 
 /**
  * ```{note} There is no Access Control Policies specification yet. As such, this
  * function is still experimental and subject to change, even in a non-major release.
  * ```
  *
- * Initialise a new empty [[SolidDataset]] to story [[AccessPolicy]]'s in.
+ * Initialise a new empty [[SolidDataset]] to story [[Policy]]'s in.
```

Comment:
Typo - 'story' should be 'store'. (Don't really need the word 'in' at the end of the sentence either.)